lib/item_handler_records.rb

```diff
@@ -0,0 +1,37 @@
+class ItemHandlerRecords
+
+  attr_accessor :records
+
+  def initialize(records_array)
+    @records = records_array
+  end
+
+  def randomize_records!
+    self.records = PreProcessingRandomizationUtil.send(ENV['RANDOMIZATION_METHOD'], records)
```

Comment:
In Ruby, do we need to require `PreProcessingRandomizationUtil` in the file if we want to use it?

Reply:
It should be fine since we are requiring the entire randomization_util

Reply:
After discussing with Kang, I've changed this